Framework/Algorithms/src/MaxEnt.cpp

```diff
@@ -20,6 +23,20 @@ using namespace Kernel;
 // Register the algorithm into the AlgorithmFactory
 DECLARE_ALGORITHM(MaxEnt)
 
+namespace {
+
+std::map<std::string, std::string> inverseCaption = {{"Time", "Frequency"},
```

Comment:
Could you add code comment to what these are please. Also in rst description add comment that algorithm will re-label units where it can according to the those you list. I think this is a nice extra feature you have added and will be useful to include in rst and in release notes (although you may argue that it would be better if the FFT algorithm itself could do this). In a future issue perhaps include an anstroom to inverse anstroem usage example. Are you testing the label conversions?